crates/kubelet/src/logs.rs

```diff
@@ -35,31 +36,44 @@ impl std::error::Error for LogSendError {
     }
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+/// Client options for fetching logs.
+pub struct LogOptions {
+    tail_lines: Option<usize>,
+    follow: Option<bool>,
```

Comment:
Can we annotate this as being false if not present on deserializing it? I'd rather just leave this a struct with public fields rather than have the getting methods

Reply:
Ok, will do.